engine/Shopware/Core/sBasket.php

```diff
@@ -259,6 +259,11 @@ public function sCheckBasketQuantities()
             }
         }
 
+        $products = $this->eventManager->filter('Shopware_Modules_Basket_CheckBasketQuantities_ProductQuantities', $products, [
```

Comment:
Please change the event name to the one below, because there are multiple products with one quantity each
```suggestion
        $products = $this->eventManager->filter('Shopware_Modules_Basket_CheckBasketQuantities_ProductsQuantity', $products, [
```